### ISSUE TITLE
command 'solargraph.config' not found

### ISSUE THREAD

**Issue by baelter** [castwide/vscode-solargraph#54]
Trying to run "Create a Solargraph config file"

```
code --version
1.23.1
d0182c3417d225529c6d5ad24b7572815d0de9ac
x64

solargraph --version
0.21.1
```

**Comment by castwide** [comment]
Confirmed. I'll fast-track a patch to correct the error.

In the meantime, you should be able to run `solargraph config` from the command line.

**Comment by castwide** [comment]
Fixed in extension 0.17.3.
